client/src/assets/components/subcomponents/NotFound.jsx

```diff
@@ -0,0 +1,20 @@
+// src/pages/NotFound.jsx
+import React from "react";
+import { Link } from "react-router-dom";
+
+const NotFound = () => {
+  return (
+    <div className="flex flex-col items-center justify-center min-h-screen text-center px-4 bg-base-200">
```

Comment:
This came out great, we recently added the Unauthorized component as well, I'd just love to keep consistentency.

Please remove the min-h-screen for now.